tests/test_blend.py

```diff
@@ -37,6 +37,8 @@ def init_data(shape, coords, amplitudes=None, convolve=True):
 
         psfs = np.array([scarlet.psf.generate_psf_image(scarlet.psf.gaussian, psf_shape, psf_center,
                                                         amplitude=1, sigma=1+.2*b) for b in range(B)])
+        psfs /= psfs.sum(axis=(1,2))[:,None,None]
```

Comment:
This is the line that breaks the other two tests. The problem is that `images` is calculated using the non-normalized PSFs, since we want it to have the same amplitude as the SEDs calculated above (so the psf was a gaussian with *amplitude* unity). So if you just move this line below line 44 and normalize after `images` is created then the tests pass.